Source/Manager.swift

```diff
@@ -9,6 +9,10 @@
 import SwiftyJSON
 
 public class Manager: NSURLProtocol {
+    typealias queryArtifacts = [String: [String]]?
+    typealias IDArtifacts = [Int]?
```

Comment:
1. Yep, it sure does. Oversight on my part. I'll find a good home for it.
1. Yep, they sure should! I'll update that.
1. Nope, not married to it. Would you have a preference?